demos/http/common/src/http_demo_utils.c

```diff
@@ -345,15 +345,17 @@ bool getS3ObjectFileSize( size_t * pFileSize,
     }
     else
     {
-        LogError( ( "Failed to read Content-Range header from HTTP response: Error=%s.",
-                    HTTPClient_strerror( httpStatus ) ) );
+        LogError( ( "Failed to read Content-Range header from HTTP response. Error: %s",
+                    ( httpStatus == HTTPSuccess ) ?
+                    "Unsuccessful response status code." : HTTPClient_strerror( httpStatus ) ) );
```

Comment:
This confuses me.  The log says "Unsuccessful" when the value was HTTPSuccess?

Reply:
The intention was to differentiate between a non-206 status code error and an HTTP error, as they both could lead to this statement, but after some more thought I agree that this is confusing -- I will remove these changes as Archit will be addressing the error logs in his PR (#1337).